releasenotes/notes/add-upstream-services-c4fee21ab10bc2fa.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add support to automatically track and propagate the history of sampling decisions in a distributed trace.
```

Comment:
Not sure how actionable or insightful this is for users. What should they expect to change?